src/olympia/addons/models.py

```diff
@@ -1579,13 +1579,24 @@ class AddonReviewerFlags(ModelBase):
 
 class MigratedLWT(OnChangeMixin, ModelBase):
     lightweight_theme_id = models.PositiveIntegerField()
-    getpersonas_id = models.PositiveIntegerField(db_index=True)
+    getpersonas_id = models.PositiveIntegerField()
     static_theme = models.ForeignKey(
         Addon, unique=True, related_name='migrated_from_lwt',
         on_delete=models.CASCADE)
 
     class Meta:
         db_table = 'migrated_personas'
+        indexes = [
+            LongNameIndex(
+                fields=('lightweight_theme_id',),
+                name='migrated_personas_lightweight_theme_id_fk_addons_id'),
```

Comment:
I wonder if we should actually drop this as it's not a foreign key any longer.

Reply:
We can. I suspect it's no longer necessary anyway...